### DISCUSSION TITLE
How can I create a subdomain in AWS Route53 when the parent domain is vercel?

### DISCUSSION THREAD

**Discussion by dacoten** [vercel/community#480] [Help] [answered]
Hey guys,

I have a main domain **E.g example.com** on Vercel.

For the reduced cost bandwidth, I want to create a subdomain ( **sub.example.com**)  on  AWS Route53. Then I point the subdomain to a static website on an S3 bucket.

How can I trigger an AWS subdomain to the main domain on Vercel?

Please help me. Thanks

**Comment by samcx** [comment, marked as answer]
You'll need to configure this on their dashboard and add a CNAME record in our DNS UI → https://docs.aws.amazon.com/amplify/latest/userguide/to-add-a-custom-domain-managed-by-a-third-party-dns-provider.html.
